molo/profiles/templates/profiles/register.html

```diff
@@ -4,48 +4,54 @@
 
 
 {% block content %}
-<div class="title profile">
-    <h3>{% trans "Join" %}</h3>
-</div>
-<div class="block">
-  <div class="signin-form">
-    <p>{% trans "Not a member yet? Join us by signing up below." %}</p>
+  <div class="super-heading l-indent">
```

Comment:
Lets just use an `h1` tag here, remove the nesting, and name the class `heading heading--profiles`